add RELEASE.md (#22)
mundialis/github-workflows: RELEASE.md

# Release and version

Steps when releasing:

- Run in terminal
  ```

>   ESTIMATED_VERSION=1.0.4
> 
>   gh api repos/mundialis/github-workflows/releases/generate-notes -f tag_name="$ESTIMATED_VERSION" -f target_commitish=main -q .body

Reviewer: ```suggestion
  ESTIMATED_VERSION=1.0.4
  REPO_NAME=mundialis/github-workflows

  gh api repos/$REPO_NAME/releases/generate-notes -f tag_name="$ESTIMATED_VERSION" -f target_commitish=main -q .body
```

  ```

> - Go to [new release](https://github.com/mundialis/github-workflows/releases/new)

Reviewer: ```suggestion
- Go to [new release](../../releases/new)
```

- Copy the output of terminal command to the release description

> - You can [compare manually](https://github.com/mundialis/github-workflows/compare/1.0.3...main) if all changes are included. If changes were pushed directly to main branch, they are not included.

Reviewer: ```suggestion
- You can [compare manually](../../compare/1.0.3...main) if all changes are included. If changes were pushed directly to main branch, they are not included.
```

- Check if `ESTIMATED_VERSION` increase still fits - we follow [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
- Fill in tag and release title with this version
- At the bottom of the release, add

>   "generated with `gh api repos/mundialis/github-workflows/releases/generate-notes -f tag_name="$ESTIMATED_VERSION" -f target_commitish=main -q .body`" and replace `$ESTIMATED_VERSION` with the actual version.

Reviewer: ```suggestion
  "generated with `gh api repos/$REPO_NAME/releases/generate-notes -f tag_name="$ESTIMATED_VERSION" -f target_commitish=main -q .body`" and replace `$REPO_NAME` and `$ESTIMATED_VERSION` with the actual version.
```
